examples/windows_ip.py

```diff
@@ -0,0 +1,5 @@
+from pyinfra import host
+
+# print ip address for all network entries
+for index in host.fact.windows_network_configuration['Index']:
```

Comment:
This could be condensted into:

```py
for index, details in host.fact.windows_network_configuration:
    print(details['IPAddress'])
```

Reply:
Any idea how to do that? There are multiple interfaces. Which one should be used?

Reply:
The above will print out the IP for each interface. Let's leave as-is for now and can play around with it once merged!